iac/environments/uat/main.tf

```diff
@@ -0,0 +1,12 @@
+#########################################################
+# Environment: UAT
+#
+# Deploy CaT resources
+#########################################################
+module "deploy-all" {
+  source      = "../../modules/configs/deploy-all"
+  space       = "uat"
+  environment = "uat"
+  cf_username = var.cf_username
+  cf_password = var.cf_password
```

Comment:
I think UAT may have memory of 1GB and instance count of 3 (according to the BoM)

Reply:
Ah yep - good spot - copied in late I think... 🤦‍♂️